ci/ray_ci/doc/cmd_check_api_discrepancy.py

```diff
@@ -70,27 +71,46 @@ def main(ray_checkout_dir: str, team: str) -> None:
     white_list_apis = TEAM_API_CONFIGS[team]["white_list_apis"]
 
     # Policy 01: all public APIs should be documented
-    print("Validating that public APIs should be documented...")
-    _validate_documented_public_apis(api_in_codes, api_in_docs, white_list_apis)
+    logger.info("Validating that public APIs should be documented...")
+    good_apis, bad_apis = _split_good_and_bad_apis(
+        api_in_codes, api_in_docs, white_list_apis
+    )
+
+    logger.info("Public APIs that are documented:")
+    for api in good_apis:
```

Comment:
python set iteration order is arbitrary / not stable.

you might want to convert to list and sort it?

Reply:
let me just return a list from the other function